src/app/properties/properties.module.ts

```diff
@@ -14,6 +14,9 @@ import { PropertyManagementModule } from './property-management/property-managem
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { MatAutocompleteModule } from '@angular/material/autocomplete';
 import { MatInputModule } from '@angular/material/input';
+import { FilterDialogComponent } from './filter-dialog/filter-dialog.component';
+import {MatSelectModule} from "@angular/material/select";
+import {MatCheckboxModule} from "@angular/material/checkbox";
```

Comment:
Do not import Angular Material modules by themselves, import Material Module.
```suggestion
    import { MaterialModule } from '../../infrastructure/material/material.module';
```